docker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:xenial
+
+RUN apt-get update && apt-get install -y \
+        curl \
+        lsb-release \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN curl -sS https://dl.google.com/cloudagents/install-logging-agent.sh | bash
+
+RUN /opt/google-fluentd/embedded/bin/gem uninstall fluent-plugin-google-cloud \
+    && /opt/google-fluentd/embedded/bin/gem install fluent-plugin-google-cloud -v 0.6.4.pre.3 \
+    && apt-get autoremove
+
+COPY google-fluentd.conf /etc/google-fluentd/google-fluentd.conf
+
+CMD sed -i "s/metadata-agent/$METADATA_AGENT_HOSTNAME/" "/etc/google-fluentd/google-fluentd.conf" \
```

Comment:
In a non-docker environment, we use `local-metadata-agent.stackdriver.com:8000` as the default value. And that is the default value in the gem.

Do we want to put that here?

I'm afraid for GKE/K8s it does not really make sense and could be misleading. Yet for raw docker, it might be relevant...